packages/input-edit/src/components/inputedittable/InputEditTable.js

```diff
@@ -0,0 +1,168 @@
+            <StyledHeader key={`${column.id}-header`}
+              onClick={() => setSorting(column.id, getDirection(sorting, column.id))}>
+              {column.label}
+            </StyledHeader>
+          )
+        }
+      </tr>
+    </thead>
+    <tbody>
+      {
+        Object.entries(data).map(([index, nodes]) =>
+          <tr key={index}>
+            <TableCells index={index}
+              nodes={nodes}
+              dataFormCells={dataFormCells}
+              inputEditForm={inputEditForm}
+              updateValue={updateValue}/>
+          </tr>
+        )
+      }
+    </tbody>
+  </StyledTable>
+}
+
+const arrowKeyHandler = listSize => event => {
+  if (document.activeElement.tagName === 'INPUT' && (event.key === 'ArrowUp' || event.key === 'ArrowDown')) {
+    const [idIndex, node] = document.activeElement.id.split(':')
+    const index = Number.parseInt(idIndex)
+    event.preventDefault()
+    const elementToFocus = getElementToFocus(index, node, listSize, event.key)
+    if (elementToFocus) {
+      elementToFocus.focus()
+      elementToFocus.select()
+    }
+  }
+}
+
+const getElementToFocus = (index, node, listSize, key) => {
+  if (key === 'ArrowUp' && index > 0) {
+    return document.getElementById(`${index - 1}:${node}`)
+  } else if (key === 'ArrowDown' && index < listSize - 1) {
+    return document.getElementById(`${index + 1}:${node}`)
+  }
+}
+
+const getDirection = ({field, direction}, newSortingField) => {
+  if (field === newSortingField) {
+    if (direction === 'asc') {
+      return 'desc'
+    } else {
+      return 'asc'
+    }
+  } else {
+    return 'asc'
+  }
+}
+
+const getDataFormCells = inputDataForm => {
+  if (Object.keys(inputDataForm).length === 0) {
+    return []
+  }
+
+  const table = inputDataForm.children.find(child => child.componentType === 'table')
+  if (Array.isArray(table.children)) {
+    return table.children
+  }
+
+  consoleLogger.logError('InputEditTable did not receive the expected schema for the list form')
+  return []
+}
+
+const keyBuilder = (index, column) => `${index}:${column.id}`
+
+const TableCells = ({index, nodes, dataFormCells, inputEditForm, updateValue}) => {
+  const dataCells = dataFormCells.map(column =>
+    <DataCell key={keyBuilder(index, column)} index={index} nodes={nodes} column={column}/>
+  )
+  const editCells = inputEditForm.map(column =>
+    <InputCell key={keyBuilder(index, column)} index={index} nodes={nodes} column={column} updateValue={updateValue}/>
+  )
+  return [...dataCells, ...editCells]
+}
+
+const DataCell = ({index, nodes, column}) => {
+  const field = column.children[0]
+  return <td>
+    {formField.formattedValueFactory(field.dataType)(
+      field, undefined, 'Input_edit_data_list', nodes[field.path], {}, {}, {}, `${index}:${column.id}`
+    )}
+  </td>
+}
+
+const InputCell = ({index, nodes, column, updateValue}) => {
+  const width = column.dataType === 'single-select'
```

Comment:
:laughing:

Reply:
please check this with yilmaz. As a temporary workaround until the layout is a bit more mature we can leave it :)

Reply:
Will do.
Sorry, I meant to warn you about this "special" piece of code :P As mentioned today, a surprising amount of people say this is the way to do it, but I'm sure there is a better way.